metricbeat/docs/modules/http.asciidoc

```diff
@@ -38,6 +38,7 @@ metricbeat.modules:
   #request.enabled: false
   #response.enabled: false
   #dedot.enabled: false
+  #response.isarray: false
```

Comment:
I'm thinking if we should make this `json.is_array` or `json.response.is_array`. Now that we have multiple metricsets in the http module not all settings apply to all metricsets I think.

I'm aware that previous configs do not follow the convention :-(